src/components/videos-you-might-like/videos.js

```diff
@@ -0,0 +1,80 @@
+export default [
```

Comment:
For improved debugging, can we name this array?
```suggestion
export const videos = [
```